drone-bom/pom.xml

```diff
@@ -39,9 +39,10 @@
   </developers>
 
   <properties>
-    <version.org.jboss.arquillian.selenium>3.11.0</version.org.jboss.arquillian.selenium>
-    <version.htmlunit.driver>2.28</version.htmlunit.driver>
-    <phantomjs.driver.version>1.4.4</phantomjs.driver.version>
+    <version.selenium>4.0.0</version.selenium>
+    <version.selenium.server>4.0.0-alpha-2</version.selenium.server>
```

Comment:
This is the last release when they released `selenium-server-standalone.jar`

Moving forward (`4.1.0` at this point) there's a single command which has `standalone` subcommand to use. See https://github.com/arquillian/arquillian-extension-drone/pull/273#issuecomment-977198261